### ISSUE TITLE
translations from two keys visible as one, placed in one field

### ISSUE THREAD

**Issue by rajaaa92** [tolk/tolk#103]
When adding a new locale, we get a lot of phrases to translate. One of them is the one with the key `activerecord.errors.messages.restrict_dependent_destroy`. It looks like this in the english version (visible on tolk):
```ruby
---
one: Cannot delete record because a dependent %{record} exists
many: Cannot delete record because dependent %{record} exist
```
In my app I ask people not related to programming to translate the app. So if they see this phrase, they start to translate the "one" and "many", forget about the "---" etc. And when clicking on Save, they get an ugly error (Psych::SyntaxError: (<unknown>): mapping values are not allowed in this context). Would it be possible, to split this phrase into 2 separate phrases, with two separate keys? So that for the key `activerecord.errors.messages.restrict_dependent_destroy.one` they would have to translate `Cannot delete record because a dependent %{record} exists` and the second by analogy.

I know those phrases are already translated on the internet and I could add them to the app. I will do, but sometimes they add a new locale, let's say, for fun, without telling me, so I don't add the translations and I don't run the import for all the locales in the world if I'm not asked to. Still, I would like to avoid seeing the error.

Thanks!

**Comment by mauriciopasquier** [comment]
Yeah, I find this really weird. Why are these keys treated differently? They should be 2 keys:

    activerecord.errors.messages.restrict_dependent_destroy.one
    activerecord.errors.messages.restrict_dependent_destroy.other

**Comment by nAzgalor** [comment]
because there is PLURALIZATION in the code

**Comment by nAzgalor** [comment]
I have another error:
`Psych::SyntaxError ((<unknown>): could not find expected directive name while scanning a directive at line 1 column 1):`

example the same.
```
---
one: Cannot delete record because a dependent %{record} exists
many: Cannot delete record because dependent %{record} exist
```
When I add only one translation, for example: 
`%{record} `
I will get this error
`Psych::SyntaxError ((<unknown>): could not find expected directive name while scanning a directive at line 1 column 1):`

**Comment by marcindrozd** [comment]
Has anyone was able to solve this?
